Description: Upstream changes introduced in version 1:1.2.1-1.1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 nfs-utils (1:1.2.1-1.1) unstable; urgency=low
 .
   * Non-maintainer upload.
   * Disable tirpc (Closes: #562729, #562757, #562910)
 .
 The person named in the Author field signed this changelog entry.
Author: Alexander Wirt <formorer@debian.org>
Bug-Debian: http://bugs.debian.org/562729
Bug-Debian: http://bugs.debian.org/562757
Bug-Debian: http://bugs.debian.org/562910

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- /dev/null
+++ nfs-utils-1.2.1/config.log
@@ -0,0 +1,694 @@
+This file contains any messages produced by compilers while
+running configure, to aid debugging if configure makes a mistake.
+
+It was created by linux nfs-utils configure 1.2.1, which was
+generated by GNU Autoconf 2.63.  Invocation command line was
+
+  $ ./configure --mandir=${prefix}/share/man --disable-tirpc --with-tcp-wrappers
+
+## --------- ##
+## Platform. ##
+## --------- ##
+
+hostname = lisa
+uname -m = x86_64
+uname -r = 2.6.32-trunk-amd64
+uname -s = Linux
+uname -v = #1 SMP Sat Dec 26 17:13:29 UTC 2009
+
+/usr/bin/uname -p = unknown
+/bin/uname -X     = unknown
+
+/bin/arch              = unknown
+/usr/bin/arch -k       = unknown
+/usr/convex/getsysinfo = unknown
+/usr/bin/hostinfo      = unknown
+/bin/machine           = unknown
+/usr/bin/oslevel       = unknown
+/bin/universe          = unknown
+
+PATH: /usr/sbin
+PATH: /usr/bin
+PATH: /sbin
+PATH: /bin
+PATH: /usr/bin/X11
+
+
+## ----------- ##
+## Core tests. ##
+## ----------- ##
+
+configure:2191: checking build system type
+configure:2209: result: x86_64-unknown-linux-gnu
+configure:2231: checking host system type
+configure:2246: result: x86_64-unknown-linux-gnu
+configure:2285: checking for a BSD-compatible install
+configure:2353: result: /usr/bin/install -c
+configure:2364: checking whether build environment is sane
+configure:2424: result: yes
+configure:2565: checking for a thread-safe mkdir -p
+configure:2604: result: /bin/mkdir -p
+configure:2617: checking for gawk
+configure:2633: found /usr/bin/gawk
+configure:2644: result: gawk
+configure:2655: checking whether make sets $(MAKE)
+configure:2677: result: yes
+configure:2762: checking whether to enable maintainer-specific portions of Makefiles
+configure:2771: result: no
+configure:3061: checking for style of include used by make
+configure:3089: result: GNU
+configure:3159: checking for gcc
+configure:3175: found /usr/bin/gcc
+configure:3186: result: gcc
+configure:3418: checking for C compiler version
+configure:3426: gcc --version >&5
+gcc (Debian 4.4.2-8) 4.4.2
+Copyright (C) 2009 Free Software Foundation, Inc.
+This is free software; see the source for copying conditions.  There is NO
+warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+
+configure:3430: $? = 0
+configure:3437: gcc -v >&5
+Using built-in specs.
+Target: x86_64-linux-gnu
+Configured with: ../src/configure -v --with-pkgversion='Debian 4.4.2-8' --with-bugurl=file:///usr/share/doc/gcc-4.4/README.Bugs --enable-languages=c,c++,fortran,objc,obj-c++ --prefix=/usr --enable-shared --enable-multiarch --enable-linker-build-id --with-system-zlib --libexecdir=/usr/lib --without-included-gettext --enable-threads=posix --with-gxx-include-dir=/usr/include/c++/4.4 --program-suffix=-4.4 --enable-nls --enable-clocale=gnu --enable-libstdcxx-debug --enable-objc-gc --with-arch-32=i486 --with-tune=generic --enable-checking=release --build=x86_64-linux-gnu --host=x86_64-linux-gnu --target=x86_64-linux-gnu
+Thread model: posix
+gcc version 4.4.2 (Debian 4.4.2-8) 
+configure:3441: $? = 0
+configure:3448: gcc -V >&5
+gcc: '-V' option must have argument
+configure:3452: $? = 1
+configure:3475: checking for C compiler default output file name
+configure:3497: gcc -g -Wall -O2   conftest.c  >&5
+configure:3501: $? = 0
+configure:3539: result: a.out
+configure:3558: checking whether the C compiler works
+configure:3568: ./a.out
+configure:3572: $? = 0
+configure:3591: result: yes
+configure:3598: checking whether we are cross compiling
+configure:3600: result: no
+configure:3603: checking for suffix of executables
+configure:3610: gcc -o conftest -g -Wall -O2   conftest.c  >&5
+configure:3614: $? = 0
+configure:3640: result: 
+configure:3646: checking for suffix of object files
+configure:3672: gcc -c -g -Wall -O2  conftest.c >&5
+configure:3676: $? = 0
+configure:3701: result: o
+configure:3705: checking whether we are using the GNU C compiler
+configure:3734: gcc -c -g -Wall -O2  conftest.c >&5
+configure:3741: $? = 0
+configure:3758: result: yes
+configure:3767: checking whether gcc accepts -g
+configure:3797: gcc -c -g  conftest.c >&5
+configure:3804: $? = 0
+configure:3905: result: yes
+configure:3922: checking for gcc option to accept ISO C89
+configure:3996: gcc  -c -g -Wall -O2  conftest.c >&5
+configure:4003: $? = 0
+configure:4026: result: none needed
+configure:4046: checking dependency style of gcc
+configure:4156: result: gcc3
+configure:4177: checking how to run the C preprocessor
+configure:4217: gcc -E  conftest.c
+configure:4224: $? = 0
+configure:4255: gcc -E  conftest.c
+conftest.c:14:28: error: ac_nonexistent.h: No such file or directory
+configure:4262: $? = 1
+configure: failed program was:
+| /* confdefs.h.  */
+| #define PACKAGE_NAME "linux nfs-utils"
+| #define PACKAGE_TARNAME "nfs-utils"
+| #define PACKAGE_VERSION "1.2.1"
+| #define PACKAGE_STRING "linux nfs-utils 1.2.1"
+| #define PACKAGE_BUGREPORT "linux-nfs@vger.kernel.org"
+| #define PACKAGE "nfs-utils"
+| #define VERSION "1.2.1"
+| #define START_STATD "/usr/sbin/start-statd"
+| #define NFS3_SUPPORTED 1
+| #define NFS4_SUPPORTED 1
+| #define GSS_SUPPORTED 1
+| /* end confdefs.h.  */
+| #include <ac_nonexistent.h>
+configure:4295: result: gcc -E
+configure:4324: gcc -E  conftest.c
+configure:4331: $? = 0
+configure:4362: gcc -E  conftest.c
+conftest.c:14:28: error: ac_nonexistent.h: No such file or directory
+configure:4369: $? = 1
+configure: failed program was:
+| /* confdefs.h.  */
+| #define PACKAGE_NAME "linux nfs-utils"
+| #define PACKAGE_TARNAME "nfs-utils"
+| #define PACKAGE_VERSION "1.2.1"
+| #define PACKAGE_STRING "linux nfs-utils 1.2.1"
+| #define PACKAGE_BUGREPORT "linux-nfs@vger.kernel.org"
+| #define PACKAGE "nfs-utils"
+| #define VERSION "1.2.1"
+| #define START_STATD "/usr/sbin/start-statd"
+| #define NFS3_SUPPORTED 1
+| #define NFS4_SUPPORTED 1
+| #define GSS_SUPPORTED 1
+| /* end confdefs.h.  */
+| #include <ac_nonexistent.h>
+configure:4409: checking for grep that handles long lines and -e
+configure:4469: result: /bin/grep
+configure:4474: checking for egrep
+configure:4538: result: /bin/grep -E
+configure:4543: checking for ANSI C header files
+configure:4573: gcc -c -g -Wall -O2  conftest.c >&5
+configure:4580: $? = 0
+configure:4679: gcc -o conftest -g -Wall -O2   conftest.c  >&5
+configure:4683: $? = 0
+configure:4689: ./conftest
+configure:4693: $? = 0
+configure:4711: result: yes
+configure:4735: checking for sys/types.h
+configure:4756: gcc -c -g -Wall -O2  conftest.c >&5
+configure:4763: $? = 0
+configure:4780: result: yes
+configure:4735: checking for sys/stat.h
+configure:4756: gcc -c -g -Wall -O2  conftest.c >&5
+configure:4763: $? = 0
+configure:4780: result: yes
+configure:4735: checking for stdlib.h
+configure:4756: gcc -c -g -Wall -O2  conftest.c >&5
+configure:4763: $? = 0
+configure:4780: result: yes
+configure:4735: checking for string.h
+configure:4756: gcc -c -g -Wall -O2  conftest.c >&5
+configure:4763: $? = 0
+configure:4780: result: yes
+configure:4735: checking for memory.h
+configure:4756: gcc -c -g -Wall -O2  conftest.c >&5
+configure:4763: $? = 0
+configure:4780: result: yes
+configure:4735: checking for strings.h
+configure:4756: gcc -c -g -Wall -O2  conftest.c >&5
+configure:4763: $? = 0
+configure:4780: result: yes
+configure:4735: checking for inttypes.h
+configure:4756: gcc -c -g -Wall -O2  conftest.c >&5
+configure:4763: $? = 0
+configure:4780: result: yes
+configure:4735: checking for stdint.h
+configure:4756: gcc -c -g -Wall -O2  conftest.c >&5
+configure:4763: $? = 0
+configure:4780: result: yes
+configure:4735: checking for unistd.h
+configure:4756: gcc -c -g -Wall -O2  conftest.c >&5
+configure:4763: $? = 0
+configure:4780: result: yes
+configure:5100: checking for libwrap
+configure:5126: gcc -o conftest -g -Wall -O2   conftest.c -lwrap  >&5
+configure:5133: $? = 0
+configure:5142: result: yes
+configure:5192: checking for special C compiler options needed for large files
+configure:5287: result: no
+configure:5293: checking for _FILE_OFFSET_BITS value needed for large files
+configure:5328: gcc -c -g -Wall -O2  conftest.c >&5
+configure:5335: $? = 0
+configure:5404: result: no
+configure:5605: checking for g++
+configure:5621: found /usr/bin/g++
+configure:5632: result: g++
+configure:5659: checking for C++ compiler version
+configure:5667: g++ --version >&5
+g++ (Debian 4.4.2-8) 4.4.2
+Copyright (C) 2009 Free Software Foundation, Inc.
+This is free software; see the source for copying conditions.  There is NO
+warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+
+configure:5671: $? = 0
+configure:5678: g++ -v >&5
+Using built-in specs.
+Target: x86_64-linux-gnu
+Configured with: ../src/configure -v --with-pkgversion='Debian 4.4.2-8' --with-bugurl=file:///usr/share/doc/gcc-4.4/README.Bugs --enable-languages=c,c++,fortran,objc,obj-c++ --prefix=/usr --enable-shared --enable-multiarch --enable-linker-build-id --with-system-zlib --libexecdir=/usr/lib --without-included-gettext --enable-threads=posix --with-gxx-include-dir=/usr/include/c++/4.4 --program-suffix=-4.4 --enable-nls --enable-clocale=gnu --enable-libstdcxx-debug --enable-objc-gc --with-arch-32=i486 --with-tune=generic --enable-checking=release --build=x86_64-linux-gnu --host=x86_64-linux-gnu --target=x86_64-linux-gnu
+Thread model: posix
+gcc version 4.4.2 (Debian 4.4.2-8) 
+configure:5682: $? = 0
+configure:5689: g++ -V >&5
+g++: '-V' option must have argument
+configure:5693: $? = 1
+configure:5696: checking whether we are using the GNU C++ compiler
+configure:5725: g++ -c -g -O2  conftest.cpp >&5
+configure:5732: $? = 0
+configure:5749: result: yes
+configure:5758: checking whether g++ accepts -g
+configure:5788: g++ -c -g  conftest.cpp >&5
+configure:5795: $? = 0
+configure:5896: result: yes
+configure:5921: checking dependency style of g++
+configure:6031: result: gcc3
+configure:6094: checking for gcc
+configure:6121: result: gcc
+configure:6353: checking for C compiler version
+configure:6361: gcc --version >&5
+gcc (Debian 4.4.2-8) 4.4.2
+Copyright (C) 2009 Free Software Foundation, Inc.
+This is free software; see the source for copying conditions.  There is NO
+warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+
+configure:6365: $? = 0
+configure:6372: gcc -v >&5
+Using built-in specs.
+Target: x86_64-linux-gnu
+Configured with: ../src/configure -v --with-pkgversion='Debian 4.4.2-8' --with-bugurl=file:///usr/share/doc/gcc-4.4/README.Bugs --enable-languages=c,c++,fortran,objc,obj-c++ --prefix=/usr --enable-shared --enable-multiarch --enable-linker-build-id --with-system-zlib --libexecdir=/usr/lib --without-included-gettext --enable-threads=posix --with-gxx-include-dir=/usr/include/c++/4.4 --program-suffix=-4.4 --enable-nls --enable-clocale=gnu --enable-libstdcxx-debug --enable-objc-gc --with-arch-32=i486 --with-tune=generic --enable-checking=release --build=x86_64-linux-gnu --host=x86_64-linux-gnu --target=x86_64-linux-gnu
+Thread model: posix
+gcc version 4.4.2 (Debian 4.4.2-8) 
+configure:6376: $? = 0
+configure:6383: gcc -V >&5
+gcc: '-V' option must have argument
+configure:6387: $? = 1
+configure:6390: checking whether we are using the GNU C compiler
+configure:6443: result: yes
+configure:6452: checking whether gcc accepts -g
+configure:6590: result: yes
+configure:6607: checking for gcc option to accept ISO C89
+configure:6711: result: none needed
+configure:6731: checking dependency style of gcc
+configure:6841: result: gcc3
+configure:6861: checking how to run the C preprocessor
+configure:6979: result: gcc -E
+configure:7008: gcc -E  conftest.c
+configure:7015: $? = 0
+configure:7046: gcc -E  conftest.c
+conftest.c:27:28: error: ac_nonexistent.h: No such file or directory
+configure:7053: $? = 1
+configure: failed program was:
+| /* confdefs.h.  */
+| #define PACKAGE_NAME "linux nfs-utils"
+| #define PACKAGE_TARNAME "nfs-utils"
+| #define PACKAGE_VERSION "1.2.1"
+| #define PACKAGE_STRING "linux nfs-utils 1.2.1"
+| #define PACKAGE_BUGREPORT "linux-nfs@vger.kernel.org"
+| #define PACKAGE "nfs-utils"
+| #define VERSION "1.2.1"
+| #define START_STATD "/usr/sbin/start-statd"
+| #define NFS3_SUPPORTED 1
+| #define NFS4_SUPPORTED 1
+| #define GSS_SUPPORTED 1
+| #define STDC_HEADERS 1
+| #define HAVE_SYS_TYPES_H 1
+| #define HAVE_SYS_STAT_H 1
+| #define HAVE_STDLIB_H 1
+| #define HAVE_STRING_H 1
+| #define HAVE_MEMORY_H 1
+| #define HAVE_STRINGS_H 1
+| #define HAVE_INTTYPES_H 1
+| #define HAVE_STDINT_H 1
+| #define HAVE_UNISTD_H 1
+| #define LIBWRAP 1
+| #define HAVE_LIBWRAP 1
+| #define HAVE_TCP_WRAPPER 1
+| /* end confdefs.h.  */
+| #include <ac_nonexistent.h>
+configure:7106: checking for a BSD-compatible install
+configure:7174: result: /usr/bin/install -c
+configure:7185: checking whether ln -s works
+configure:7189: result: yes
+configure:7196: checking whether make sets $(MAKE)
+configure:7218: result: yes
+configure:7252: checking for a sed that does not truncate output
+configure:7318: result: /bin/sed
+configure:7336: checking for fgrep
+configure:7400: result: /bin/grep -F
+configure:7435: checking for ld used by gcc
+configure:7502: result: /usr/bin/ld
+configure:7511: checking if the linker (/usr/bin/ld) is GNU ld
+configure:7526: result: yes
+configure:7538: checking for BSD- or MS-compatible name lister (nm)
+configure:7587: result: /usr/bin/nm -B
+configure:7705: checking the name lister (/usr/bin/nm -B) interface
+configure:7712: gcc -c -g -Wall -O2  conftest.c >&5
+configure:7715: /usr/bin/nm -B "conftest.o"
+configure:7718: output
+0000000000000000 B some_variable
+configure:7725: result: BSD nm
+configure:7729: checking the maximum length of command line arguments
+configure:7849: result: 3458764513820540925
+configure:7866: checking whether the shell understands some XSI constructs
+configure:7876: result: yes
+configure:7880: checking whether the shell understands "+="
+configure:7886: result: yes
+configure:7921: checking for /usr/bin/ld option to reload object files
+configure:7928: result: -r
+configure:7997: checking for objdump
+configure:8013: found /usr/bin/objdump
+configure:8024: result: objdump
+configure:8056: checking how to recognize dependent libraries
+configure:8252: result: pass_all
+configure:8312: checking for ar
+configure:8328: found /usr/bin/ar
+configure:8339: result: ar
+configure:8417: checking for strip
+configure:8433: found /usr/bin/strip
+configure:8444: result: strip
+configure:8516: checking for ranlib
+configure:8532: found /usr/bin/ranlib
+configure:8543: result: ranlib
+configure:8633: checking command to parse /usr/bin/nm -B output from gcc object
+configure:8751: gcc -c -g -Wall -O2  conftest.c >&5
+configure:8754: $? = 0
+configure:8758: /usr/bin/nm -B conftest.o \| sed -n -e 's/^.*[	 ]\([ABCDGIRSTW][ABCDGIRSTW]*\)[	 ][	 ]*\([_A-Za-z][_A-Za-z0-9]*\)$/\1 \2 \2/p' \> conftest.nm
+configure:8761: $? = 0
+configure:8815: gcc -o conftest -g -Wall -O2   conftest.c conftstm.o >&5
+configure:8818: $? = 0
+configure:8856: result: ok
+configure:8951: gcc -c -g -Wall -O2  conftest.c >&5
+configure:8954: $? = 0
+configure:9721: checking for dlfcn.h
+configure:9742: gcc -c -g -Wall -O2  conftest.c >&5
+configure:9749: $? = 0
+configure:9766: result: yes
+configure:9893: checking for C++ compiler version
+configure:9901: g++ --version >&5
+g++ (Debian 4.4.2-8) 4.4.2
+Copyright (C) 2009 Free Software Foundation, Inc.
+This is free software; see the source for copying conditions.  There is NO
+warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+
+configure:9905: $? = 0
+configure:9912: g++ -v >&5
+Using built-in specs.
+Target: x86_64-linux-gnu
+Configured with: ../src/configure -v --with-pkgversion='Debian 4.4.2-8' --with-bugurl=file:///usr/share/doc/gcc-4.4/README.Bugs --enable-languages=c,c++,fortran,objc,obj-c++ --prefix=/usr --enable-shared --enable-multiarch --enable-linker-build-id --with-system-zlib --libexecdir=/usr/lib --without-included-gettext --enable-threads=posix --with-gxx-include-dir=/usr/include/c++/4.4 --program-suffix=-4.4 --enable-nls --enable-clocale=gnu --enable-libstdcxx-debug --enable-objc-gc --with-arch-32=i486 --with-tune=generic --enable-checking=release --build=x86_64-linux-gnu --host=x86_64-linux-gnu --target=x86_64-linux-gnu
+Thread model: posix
+gcc version 4.4.2 (Debian 4.4.2-8) 
+configure:9916: $? = 0
+configure:9923: g++ -V >&5
+g++: '-V' option must have argument
+configure:9927: $? = 1
+configure:9930: checking whether we are using the GNU C++ compiler
+configure:9983: result: yes
+configure:9992: checking whether g++ accepts -g
+configure:10130: result: yes
+configure:10155: checking dependency style of g++
+configure:10265: result: gcc3
+configure:10288: checking how to run the C++ preprocessor
+configure:10324: g++ -E  conftest.cpp
+configure:10331: $? = 0
+configure:10362: g++ -E  conftest.cpp
+
+## ---------------- ##
+## Cache variables. ##
+## ---------------- ##
+
+ac_cv_build=x86_64-unknown-linux-gnu
+ac_cv_c_compiler_gnu=yes
+ac_cv_cxx_compiler_gnu=yes
+ac_cv_env_CCC_set=
+ac_cv_env_CCC_value=
+ac_cv_env_CC_set=
+ac_cv_env_CC_value=
+ac_cv_env_CFLAGS_set=set
+ac_cv_env_CFLAGS_value='-g -Wall -O2'
+ac_cv_env_CPPFLAGS_set=set
+ac_cv_env_CPPFLAGS_value=
+ac_cv_env_CPP_set=
+ac_cv_env_CPP_value=
+ac_cv_env_CXXCPP_set=
+ac_cv_env_CXXCPP_value=
+ac_cv_env_CXXFLAGS_set=set
+ac_cv_env_CXXFLAGS_value='-g -O2'
+ac_cv_env_CXX_set=
+ac_cv_env_CXX_value=
+ac_cv_env_GSSGLUE_CFLAGS_set=
+ac_cv_env_GSSGLUE_CFLAGS_value=
+ac_cv_env_GSSGLUE_LIBS_set=
+ac_cv_env_GSSGLUE_LIBS_value=
+ac_cv_env_LDFLAGS_set=set
+ac_cv_env_LDFLAGS_value=
+ac_cv_env_LIBS_set=
+ac_cv_env_LIBS_value=
+ac_cv_env_PKG_CONFIG_set=
+ac_cv_env_PKG_CONFIG_value=
+ac_cv_env_RPCSECGSS_CFLAGS_set=
+ac_cv_env_RPCSECGSS_CFLAGS_value=
+ac_cv_env_RPCSECGSS_LIBS_set=
+ac_cv_env_RPCSECGSS_LIBS_value=
+ac_cv_env_build_alias_set=
+ac_cv_env_build_alias_value=
+ac_cv_env_host_alias_set=
+ac_cv_env_host_alias_value=
+ac_cv_env_target_alias_set=
+ac_cv_env_target_alias_value=
+ac_cv_header_dlfcn_h=yes
+ac_cv_header_inttypes_h=yes
+ac_cv_header_memory_h=yes
+ac_cv_header_stdc=yes
+ac_cv_header_stdint_h=yes
+ac_cv_header_stdlib_h=yes
+ac_cv_header_string_h=yes
+ac_cv_header_strings_h=yes
+ac_cv_header_sys_stat_h=yes
+ac_cv_header_sys_types_h=yes
+ac_cv_header_unistd_h=yes
+ac_cv_host=x86_64-unknown-linux-gnu
+ac_cv_objext=o
+ac_cv_path_EGREP='/bin/grep -E'
+ac_cv_path_FGREP='/bin/grep -F'
+ac_cv_path_GREP=/bin/grep
+ac_cv_path_SED=/bin/sed
+ac_cv_path_install='/usr/bin/install -c'
+ac_cv_path_mkdir=/bin/mkdir
+ac_cv_prog_AWK=gawk
+ac_cv_prog_CPP='gcc -E'
+ac_cv_prog_ac_ct_AR=ar
+ac_cv_prog_ac_ct_CC=gcc
+ac_cv_prog_ac_ct_CXX=g++
+ac_cv_prog_ac_ct_OBJDUMP=objdump
+ac_cv_prog_ac_ct_RANLIB=ranlib
+ac_cv_prog_ac_ct_STRIP=strip
+ac_cv_prog_cc_c89=
+ac_cv_prog_cc_g=yes
+ac_cv_prog_cxx_g=yes
+ac_cv_prog_make_make_set=yes
+ac_cv_sys_file_offset_bits=no
+ac_cv_sys_largefile_CC=no
+am_cv_CC_dependencies_compiler_type=gcc3
+am_cv_CXX_dependencies_compiler_type=gcc3
+lt_cv_deplibs_check_method=pass_all
+lt_cv_file_magic_cmd='$MAGIC_CMD'
+lt_cv_file_magic_test_file=
+lt_cv_ld_reload_flag=-r
+lt_cv_nm_interface='BSD nm'
+lt_cv_path_LD=/usr/bin/ld
+lt_cv_path_NM='/usr/bin/nm -B'
+lt_cv_prog_gnu_ld=yes
+lt_cv_sys_global_symbol_pipe='sed -n -e '\''s/^.*[	 ]\([ABCDGIRSTW][ABCDGIRSTW]*\)[	 ][	 ]*\([_A-Za-z][_A-Za-z0-9]*\)$/\1 \2 \2/p'\'''
+lt_cv_sys_global_symbol_to_c_name_address='sed -n -e '\''s/^: \([^ ]*\) $/  {\"\1\", (void *) 0},/p'\'' -e '\''s/^[ABCDGIRSTW]* \([^ ]*\) \([^ ]*\)$/  {"\2", (void *) \&\2},/p'\'''
+lt_cv_sys_global_symbol_to_c_name_address_lib_prefix='sed -n -e '\''s/^: \([^ ]*\) $/  {\"\1\", (void *) 0},/p'\'' -e '\''s/^[ABCDGIRSTW]* \([^ ]*\) \(lib[^ ]*\)$/  {"\2", (void *) \&\2},/p'\'' -e '\''s/^[ABCDGIRSTW]* \([^ ]*\) \([^ ]*\)$/  {"lib\2", (void *) \&\2},/p'\'''
+lt_cv_sys_global_symbol_to_cdecl='sed -n -e '\''s/^T .* \(.*\)$/extern int \1();/p'\'' -e '\''s/^[ABCDGIRSTW]* .* \(.*\)$/extern char \1;/p'\'''
+lt_cv_sys_max_cmd_len=3458764513820540925
+
+## ----------------- ##
+## Output variables. ##
+## ----------------- ##
+
+ACLOCAL='${SHELL} /home/formorer/debian/nmu/nfs-utils-1.2.1/missing --run aclocal-1.11'
+ACLOCAL_AMFLAGS=''
+ALLOCA=''
+AMDEPBACKSLASH='\'
+AMDEP_FALSE='#'
+AMDEP_TRUE=''
+AMTAR='${SHELL} /home/formorer/debian/nmu/nfs-utils-1.2.1/missing --run tar'
+AM_CFLAGS=''
+AM_CPPFLAGS=''
+AR='ar'
+AUTOCONF='${SHELL} /home/formorer/debian/nmu/nfs-utils-1.2.1/missing --run autoconf'
+AUTOHEADER='${SHELL} /home/formorer/debian/nmu/nfs-utils-1.2.1/missing --run autoheader'
+AUTOMAKE='${SHELL} /home/formorer/debian/nmu/nfs-utils-1.2.1/missing --run automake-1.11'
+AWK='gawk'
+CC='gcc'
+CCDEPMODE='depmode=gcc3'
+CC_FOR_BUILD=''
+CFLAGS='-g -Wall -O2'
+CFLAGS_FOR_BUILD=''
+CONFIG_GSS_FALSE='#'
+CONFIG_GSS_TRUE=''
+CONFIG_IPV6_FALSE=''
+CONFIG_IPV6_TRUE='#'
+CONFIG_MOUNT_FALSE='#'
+CONFIG_MOUNT_TRUE=''
+CONFIG_NFSV4_FALSE='#'
+CONFIG_NFSV4_TRUE=''
+CONFIG_RPCGEN_FALSE=''
+CONFIG_RPCGEN_TRUE='#'
+CPP='gcc -E'
+CPPFLAGS=''
+CPPFLAGS_FOR_BUILD=''
+CXX='g++'
+CXXCPP='g++ -E'
+CXXDEPMODE='depmode=gcc3'
+CXXFLAGS='-g -O2'
+CXXFLAGS_FOR_BUILD=''
+CYGPATH_W='echo'
+DEFS=''
+DEPDIR='.deps'
+DSYMUTIL=''
+DUMPBIN=''
+ECHO_C=''
+ECHO_N='-n'
+ECHO_T=''
+EGREP='/bin/grep -E'
+EXEEXT=''
+FGREP='/bin/grep -F'
+GREP='/bin/grep'
+GSSD='gssd'
+GSSGLUE_CFLAGS=''
+GSSGLUE_LIBS=''
+HAVE_LIBWRAP=''
+HAVE_TCP_WRAPPER=''
+IDMAPD='idmapd'
+INSTALL_DATA='${INSTALL} -m 644'
+INSTALL_PROGRAM='${INSTALL}'
+INSTALL_SCRIPT='${INSTALL}'
+INSTALL_STRIP_PROGRAM='$(install_sh) -c -s'
+K5VERS=''
+KRBCFLAGS=''
+KRBDIR=''
+KRBLDFLAGS=''
+KRBLIBS=''
+LD='/usr/bin/ld -m elf_x86_64'
+LDFLAGS=''
+LDFLAGS_FOR_BUILD=''
+LIBBLKID=''
+LIBBSD=''
+LIBCRYPT=''
+LIBNSL=''
+LIBOBJS=''
+LIBS=''
+LIBSOCKET=''
+LIBTOOL=''
+LIBWRAP='-lwrap'
+LIPO=''
+LN_S='ln -s'
+LTLIBOBJS=''
+MAINT='#'
+MAINTAINER_MODE_FALSE=''
+MAINTAINER_MODE_TRUE='#'
+MAKEINFO='${SHELL} /home/formorer/debian/nmu/nfs-utils-1.2.1/missing --run makeinfo'
+MKDIR_P='/bin/mkdir -p'
+MOUNT_CONFIG_FALSE=''
+MOUNT_CONFIG_TRUE='#'
+NM='/usr/bin/nm -B'
+NMEDIT=''
+OBJDUMP='objdump'
+OBJEXT='o'
+OTOOL64=''
+OTOOL=''
+PACKAGE='nfs-utils'
+PACKAGE_BUGREPORT='linux-nfs@vger.kernel.org'
+PACKAGE_NAME='linux nfs-utils'
+PACKAGE_STRING='linux nfs-utils 1.2.1'
+PACKAGE_TARNAME='nfs-utils'
+PACKAGE_VERSION='1.2.1'
+PATH_SEPARATOR=':'
+PKG_CONFIG=''
+RANLIB='ranlib'
+RELEASE='1'
+RPCGEN_PATH='/usr/bin/rpcgen'
+RPCSECGSS_CFLAGS=''
+RPCSECGSS_LIBS=''
+SED='/bin/sed'
+SET_MAKE=''
+SHELL='/bin/bash'
+STRIP='strip'
+SVCGSSD='svcgssd'
+VERSION='1.2.1'
+ac_ct_CC='gcc'
+ac_ct_CXX='g++'
+ac_ct_DUMPBIN=''
+am__EXEEXT_FALSE=''
+am__EXEEXT_TRUE=''
+am__fastdepCC_FALSE='#'
+am__fastdepCC_TRUE=''
+am__fastdepCXX_FALSE='#'
+am__fastdepCXX_TRUE=''
+am__include='include'
+am__isrc=''
+am__leading_dot='.'
+am__quote=''
+am__tar='${AMTAR} chof - "$$tardir"'
+am__untar='${AMTAR} xf -'
+bindir='${exec_prefix}/bin'
+build='x86_64-unknown-linux-gnu'
+build_alias=''
+build_cpu='x86_64'
+build_os='linux-gnu'
+build_vendor='unknown'
+datadir='${datarootdir}'
+datarootdir='${prefix}/share'
+docdir='${datarootdir}/doc/${PACKAGE_TARNAME}'
+dvidir='${docdir}'
+enable_gss='yes'
+enable_ipv6=''
+enable_mountconfig=''
+enable_nfsv3='yes'
+enable_nfsv4='yes'
+exec_prefix='NONE'
+host='x86_64-unknown-linux-gnu'
+host_alias=''
+host_cpu='x86_64'
+host_os='linux-gnu'
+host_vendor='unknown'
+htmldir='${docdir}'
+includedir='${prefix}/include'
+infodir='${datarootdir}/info'
+install_sh='${SHELL} /home/formorer/debian/nmu/nfs-utils-1.2.1/install-sh'
+kprefix=''
+libdir='${exec_prefix}/lib'
+libexecdir='${exec_prefix}/libexec'
+localedir='${datarootdir}/locale'
+localstatedir='${prefix}/var'
+lt_ECHO='echo'
+mandir='${prefix}/share/man'
+mkdir_p='/bin/mkdir -p'
+mountfile=''
+oldincludedir='/usr/include'
+pdfdir='${docdir}'
+prefix='NONE'
+program_transform_name='s,x,x,'
+psdir='${docdir}'
+sbindir='${exec_prefix}/sbin'
+sharedstatedir='${prefix}/com'
+startstatd='/usr/sbin/start-statd'
+statduser='nobody'
+statedir='/var/lib/nfs'
+sysconfdir='${prefix}/etc'
+target_alias=''
+
+## ----------- ##
+## confdefs.h. ##
+## ----------- ##
+
+#define PACKAGE_NAME "linux nfs-utils"
+#define PACKAGE_TARNAME "nfs-utils"
+#define PACKAGE_VERSION "1.2.1"
+#define PACKAGE_STRING "linux nfs-utils 1.2.1"
+#define PACKAGE_BUGREPORT "linux-nfs@vger.kernel.org"
+#define PACKAGE "nfs-utils"
+#define VERSION "1.2.1"
+#define START_STATD "/usr/sbin/start-statd"
+#define NFS3_SUPPORTED 1
+#define NFS4_SUPPORTED 1
+#define GSS_SUPPORTED 1
+#define STDC_HEADERS 1
+#define HAVE_SYS_TYPES_H 1
+#define HAVE_SYS_STAT_H 1
+#define HAVE_STDLIB_H 1
+#define HAVE_STRING_H 1
+#define HAVE_MEMORY_H 1
+#define HAVE_STRINGS_H 1
+#define HAVE_INTTYPES_H 1
+#define HAVE_STDINT_H 1
+#define HAVE_UNISTD_H 1
+#define LIBWRAP 1
+#define HAVE_LIBWRAP 1
+#define HAVE_TCP_WRAPPER 1
+#define HAVE_DLFCN_H 1
+
+configure: caught signal 2
+configure: exit 1
